runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTracker.java

```diff
@@ -58,9 +63,11 @@
 	 */
 	private final Map<Tab, Long> completionTimes = new EnumMap<>(Tab.class);
 
+	private boolean newRegionLoaded;
+	private Collection<FarmingRegion> lastRegions = Collections.EMPTY_LIST;
```

Comment:
```suggestion
	private Collection<FarmingRegion> lastRegions = Collections.emptyList();
```
or I think this can be null